/**
 * This file was auto-generated by Fern from our API Definition.
 */

/**
 * This error occurs when we (Belvo) have encountered an internal system error (sorry about that) or due to an unsupported response from the institution.
 */
export interface UnexpectedError {
    /**
     * A unique error code (`unexpected_error`) that allows you to classify and handle the error programmatically.
     *
     * ℹ️ Check our DevPortal for more information on how to handle <a href="https://developers.belvo.com/docs/belvo-api-errors#500-unexpected_error" target="_blank">500 unexpected_error errors</a>.
     */
    code?: string;
    /**
     * A short description of the error.
     *
     * For `unexpected_error` errors, the description is:
     *
     *   - `Belvo is unable to process the request due to an internal system issue or to an unsupported response from an institution`.
     */
    message?: string;
    /** A 32-character unique ID of the request (matching a regex pattern of: `[a-f0-9]{32}`). Provide this ID when contacting the Belvo support team to accelerate investigations. */
    requestId?: string;
}
